packages/jest-util/src/__tests__/get_callsite.test.js

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs';
+import SourceMap from 'source-map';
+import getCallsite from '../get_callsite';
+
+jest.mock('fs');
+
+describe('getCallsite', () => {
+  beforeEach(() => {
+    fs.readFileSync = jest.fn();
```

Comment:
When you do `jest.mock` without a factory, all exports are mocks by default. So this `beforeEach` shouldn't be necessary

Reply:
You can do `jest.clearMocks` if needed

Reply:
The test also fails CI?

Reply:
Fixed, not sure what I was thinking